tensorboard/plugins/hparams/summary.py

```diff
@@ -193,5 +193,14 @@ def _summary(tag, hparams_plugin_data):
     tb_metadata = metadata.create_summary_metadata(hparams_plugin_data)
     raw_metadata = tb_metadata.SerializeToString()
     tf_metadata = tf.compat.v1.SummaryMetadata.FromString(raw_metadata)
-    summary.value.add(tag=tag, metadata=tf_metadata)
+    summary.value.add(
+        tag=tag, metadata=tf_metadata, tensor=_TF_NULL_TENSOR,
+    )
     return summary
+
+
+# Like `metadata.NULL_TENSOR`, but with the TensorFlow version of the
+# proto. Slight kludge needed to expose the `TensorProto` type.
+_TF_NULL_TENSOR = type(tf.compat.v1.Summary.Value().tensor).FromString(
```

Comment:
IMO a bit cleaner to do `type(tf.make_tensor_proto(0))` for this?

Reply:
Sure: done.